### PR TITLE
[ADD] Add service and sector to event task template and wizard

### PR DESCRIPTION
**Pull request by rimbendhaou**
Add service to event task template and wizard

### CODE REVIEW

File: project_event_template/wizard/project_event_wizard.py [hunk 1]
```diff
@@ -25,6 +25,10 @@ class ProjectEventWizard(models.TransientModel):
         'res.partner',
         string='Client',
     )
+    event_sector_id = fields.Many2one(
+        'res.partner.sector',
+        string='Faculty Sectors',
```
Comment (lgharib) [inline project_event_template/wizard/project_event_wizard.py]: We need to chagne Faculty Sectors to Faculty Sector and add French translation for this field Faculty Sector to Secteur de la factulté

File: project_event_template/wizard/project_activity_wizard.py [hunk 2]
```diff
@@ -29,6 +29,10 @@ class ProjectActivityWizard(models.TransientModel):
         'res.partner',
         string='Client',
     )
+    activity_sector_id = fields.Many2one(
+        'res.partner.sector',
+        string='Faculty Sectors',
```
Comment (lgharib) [inline project_event_template/wizard/project_activity_wizard.py]: We need to chagne Faculty Sectors to Faculty Sector and add French translation for this field Faculty Sector to Secteur de la factulté

**Review by lgharib — COMMENTED** [review]
In Activity Wizard Template Form the Faculty Sector should be inherited from the Event Wizard form and stay editable

**Review by eilst — APPROVED** [review]
Looks good

**Pull request merged**